clients/csharp-uwp/UWPVoiceAssistantSample/DirectLineSpeechDialogBackend.cs

```diff
@@ -242,7 +265,7 @@ public async Task StartAudioTurnAsync(bool performConfirmation)
         public async Task CancelSignalVerification()
 #pragma warning restore CS1998 // Async method lacks 'await' operators and will run synchronously
         {
-            _ = this.connector.StopKeywordRecognitionAsync();
+            await this.connector.StopKeywordRecognitionAsync();

```

Comment:
The only thing I recall about this call being pesky was that `StopKeywordRecognitionAsync` was sometimes taking a long time (several seconds) to complete, depending on the state of the DSC when it was called. I believe it was due to a limitation in the Carbon audio pump's state handling that made it need to time out on an operation before it'd do anything. But it may have been addressed since then -- have you seen any undue delays on this call completing? It might be worth just sanity checking the completion time.